src/utils/api.js

```diff
@@ -2,9 +2,9 @@ import axios from "axios";
 
 const api = axios.create({ baseURL: "https://nc-news-qvv1.onrender.com/api" })
 
-export const getArticles = () => {
-    return api.get('/articles')
-    .then(({ data: {articles} }) => {
+export const getArticles = (topic, sort_by, order) => {
+    return api.get(`/articles`, { params: { topic, sort_by, order } })
```

Comment:
Nice use of the axios params object! 🙌 
Great that you've refactored this function rather than making a separate API function 🚀